### PR TITLE
Dot lasher branch

### PR DESCRIPTION
**Pull request by Vidhanvyrs**
Successfully created and added the Layout component

### CODE REVIEW

File: src/components/layout/Layout.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import Header from "./Header"
+import Footer from "./Footer"
+
+const Layout = ({children}: {children: ReactNode}): JSX.Element => {
+    return (
+        <div className="relative">
```
Comment (anicmarko9) [inline src/components/layout/Layout.tsx:6]: This messes up the HTML flow. Use `<>` instead.

File: src/pages/AboutUs.tsx
Comment (anicmarko9) [inline src/pages/AboutUs.tsx:1]: In this file you have 2 `<header>`, 2 `<footer>` and 2 `<main>` tags. And in every other page that is supposed to render. I'm not really sure if you completely understood what is `<Layout>` used for.

File: src/pages/Locations.tsx
Comment (anicmarko9) [inline src/pages/Locations.tsx:1]: Same with this file, but here you have 2 headers, 2 footers and only 1 main HTML tag.

File: src/pages/WeightLossProgram.tsx
Comment (anicmarko9) [inline src/pages/WeightLossProgram.tsx:1]: Same here.

**Review by anicmarko9 — COMMENTED** [review]
Your current HTML structure looks like this:
```html
<html>
  <head>
  ...
  </head>
  <body>
    <header>
    <main>
      <header>
      <main>
      ...
      </main>
      <footer>
    </main>
    <footer>
  </body>
</html>
```

You have duplicates everywhere. I'm pretty sure you didn't run the `npm run start` command, because otherwise you would've definitely seen duplicates across the page.

File: src/pages/AboutUs.tsx
Comment (Vidhanvyrs) [reply src/pages/AboutUs.tsx:1]: Ok my bad, Resolving it ASAP

File: src/components/layout/Layout.tsx (hunk 1 above)
Comment (Vidhanvyrs) [reply src/components/layout/Layout.tsx:6]: so i can write this code 
import Header from './Header';
import Footer from './Footer';
import { ReactNode } from 'react';

const Layout = ({ children }: { children: ReactNode }): React.ReactElement => {
  return (
    <>
      <div className="relative">
        <Header />
        <main>{children}</main>
        <Footer />
      </div>
    </>
  );
};

export default Layout;

Comment (Vidhanvyrs) [reply src/components/layout/Layout.tsx:6]: you see as you told i ran the npm run start and i found this error too
ERROR in src/components/layout/Layout.tsx:4:40
TS2304: Cannot find name 'ReactNode'.
    2 | import Footer from "./Footer"
    3 |
  > 4 | const Layout = ({children}: {children: ReactNode}): JSX.Element => {
      |                                        ^^^^^^^^^
    5 |     return (
    6 |         <>
    7 |         <div className="relative">
so i corrected that too but there is another error which i was unable to figure out can you help
the Error goes like
src\components\layout\Layout.tsx
  Line 7:5:   'React' must be in scope when using JSX  react/react-in-jsx-scope
  Line 8:7:   'React' must be in scope when using JSX  react/react-in-jsx-scope
  Line 9:9:   'React' must be in scope when using JSX  react/react-in-jsx-scope
  Line 10:9:  'React' must be in scope when using JSX  react/react-in-jsx-scope
  Line 11:9:  'React' must be in scope when using JSX  react/react-in-jsx-scope

File: src/pages/WeightLossProgram.tsx
Comment (Vidhanvyrs) [reply src/pages/WeightLossProgram.tsx:1]: I resolved it

File: src/pages/Locations.tsx
Comment (Vidhanvyrs) [reply src/pages/Locations.tsx:1]: resolved this too

File: src/components/layout/Layout.tsx (hunk 1 above)
Comment (Vidhanvyrs) [reply src/components/layout/Layout.tsx:6]: okay so i was able to figure out the second error too and fixed that so just ignore the half part of the comment

**Comment by Vidhanvyrs** [comment]
Here i added all the asked changes and compared the main app with my app running on my local machine I think there are no conflicts as it is running great 
But you do Review and give me feedback if any changes required or any mistake is done 🙇‍♂️

**Comment by Vidhanvyrs** [comment]
Screenshot (164)
ya so i think it is resolved

**Comment by anicmarko9** [comment]
> Screenshot (164) ya so i think it is resolved

I'm not sure if you are messing with me on purpose or not 😂

How can you not see that you have 2 headers, 2 footers on this image that you took screenshot? You see both on the page, and inside of Inspect element.

Also, you misused the `<>` tag again, here is the documentation: [docs(<>)](https://legacy.reactjs.org/docs/fragments.html).

**Comment by Vidhanvyrs** [comment]
Sorry i was not messing with you I forgot to update the Home.tsx file 😅🤦‍♂️
here now i have udated it 
Now about the <> tag  
will this code ok 
  <>
      <Header />
      <main className="relative">{children}</main>
      <Footer />
    </>

**Comment by Vidhanvyrs** [comment]
Instead of this can i do this with the main tag 
main className="relative"

**Pull request merged**